Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM registry.fedoraproject.org/fedora:35 AS builder
+ARG BUILDARCH
+ENV BUILDARCH=${BUILDARCH}
 RUN dnf install -y golang git
 RUN mkdir /butane
 COPY . /butane
 WORKDIR /butane
 RUN ./build_for_container
 
 FROM scratch
-COPY --from=builder /butane/bin/container/butane-x86_64-unknown-linux-gnu /usr/local/bin/butane
+ARG BUILDARCH
+ENV BUILDARCH=${BUILDARCH}
```

Comment:
Shouldn't be necessary, right?

Reply:
We still need to set the build arch, as go can build arm64 on amd64.

Going to need a fee days. Had emergency surgery.

Reply:
Right, it's needed in the build stage, but I don't think it's needed in this stage.

Oof, hope you feel better soon.  There's no rush on this PR; take as much time as you need.

Reply:
I see what you mean. Will take care of it as soon as possible.

Still need to figure out how to set the manifest to allow both amd64 and arm64.